app/models/state_benefit.rb

```diff
@@ -5,7 +5,7 @@ class StateBenefit < ApplicationRecord
   belongs_to :state_benefit_type
   has_many :state_benefit_payments
 
-  delegate :exclude_from_gross_income, to: :state_benefit_type
+  delegate :exclude_from_gross_income, :exclude_from_gross_income?, to: :state_benefit_type
```

Comment:
little thing, you can ignore, I don't think this is needed ":exclude_from_gross_income" and we can keep ":exclude_from_gross_income?"